scripts/config_test_vm.psm1

```diff
@@ -567,12 +590,20 @@ function Get-LegacyRegressionTestArtifacts
         $ArtifactName = "v$ArtifactVersion/Build-x64-native-only-Release.$ArtifactVersion.zip"
         $ArtifactUrl = "https://github.com/microsoft/ebpf-for-windows/releases/download/" + $ArtifactName
 
-        $ProgressPreference = 'SilentlyContinue'
-        Invoke-WebRequest -Uri $ArtifactUrl -OutFile "$DownloadPath\artifact.zip"
-
-        Write-Log "Extracting $ArtifactName"
-        Expand-Archive -Path "$DownloadPath\artifact.zip" -DestinationPath $DownloadPath -Force
-        Expand-Archive -Path "$DownloadPath\build-NativeOnlyRelease.zip" -DestinationPath $DownloadPath -Force
+        for ($i = 0; $i -lt 5; $i++) {
+            try {
+                # Download and extract the artifact.
+                Get-ZipFileFromUrl -Url $ArtifactUrl -DownloadFilePath "$DownloadPath\artifact.zip" -OutputDir $DownloadPath
```

Comment:
nit: `Get-ZipFileFromUrl` internally also has 5 retries, and this loop also has 5 tries. Is this intentional?